codecov.yml

```diff
@@ -1,3 +1,6 @@
+fixes:
+  - "io/ballerina/stdlib/ftp/compiler/::./compiler-plugin/src/main/java/io/ballerina/stdlib/ftp/plugin/"
+
```

Comment:
Is this necessary? I havent added this to sql modules and they work

Reply:
Not sure actually. Removed it for now. Let's see the output

Reply:
Um, when I tried it did not work. But I'm not sure whether it's due to this or some other reason. Let's check the result and if it is not needed, remove it.

(Maybe this is not needed since even with the incorrect path fix, it added the coverage).

Reply:
Looks like its not needed